docs/quickstart/installing-substrate.md

```diff
@@ -28,4 +28,4 @@ This may take several minutes to finish. Once it is done, check that Substrate i
   ]
 }
 [/block]
-If you get that, then you're ready to proceed!
\ No newline at end of file
+If you get that, then you're ready to proceed!
```

Comment:
What changed on this line?

Reply:
Add an empty line at the end of the file.